compiler/rustc_trait_selection/src/traits/query/type_op/normalize.rs

```diff
@@ -81,3 +119,12 @@ impl<'tcx> Normalizable<'tcx> for ty::FnSig<'tcx> {
         tcx.type_op_normalize_fn_sig(canonicalized)
     }
 }
+
+impl<'tcx> Normalizable<'tcx> for ty::PolyTypeOutlivesPredicate<'tcx> {
```

Comment:
why do we need this impl then? :thinking:

Reply:
For the trait bounds to work out. Rust doesnt know we only call this in the new solver.

Reply:
pls add comment for this :grin: